core/cmd/remote_client.go

```diff
@@ -509,7 +509,7 @@ func (cli *Client) ConfigFileValidate(c *clipkg.Context) error {
 	if err != nil {
 		fmt.Println("Invalid configuration:", err)
 		fmt.Println()
-		return nil
+		return cli.errorOut(err)
```

Comment:
Does this double print the error now? If so, should we drop it from 510.

Reply:
Actually, maybe just print the full complex error up top, and then return a simpler error here?
```suggestion
		return cli.errorOut(errors.New("invalid configuration"))
```
Does this give us a clean one-liner at the end? e.g. `exit code 1: invalid configuration`